README.Rmd

```diff
@@ -15,58 +15,48 @@ knitr::opts_chunk$set(
+[![R build status](https://github.com/jameel-institute/daedalus.data/workflows/R-CMD-check/badge.svg)](https://github.com/jameel-institute/daedalus.data/actions/workflows/R-CMD-check.yaml)
+[![Codecov test coverage](https://codecov.io/gh/jameel-institute/daedalus.data/branch/main/graph/badge.svg)](https://app.codecov.io/gh/jameel-institute/daedalus.data?branch=main)
+[![CRAN status](https://www.r-pkg.org/badges/version/daedalus.data)](https://CRAN.R-project.org/package=daedalus.data)
 <!-- badges: end -->
 
-_ji.rpkg.template_ is a template package and repository on which future Jameel Institute packages are based.
+_daedalus.data_ is a package and repository to handle input data files for [_daedalus_](https://jameel-institute.github.io/daedalus/), Jameel Institute's combined economic-epidemiological model of emerging respiratory pandemic diseases.
 
-To use this template, select _ji.rpkg.template_ from the drop-down menu under **Repository template** when creating a new repository in the Jameel Institute organisation.
-Replace all instances of "ji.rpkg.template" with your package name. Make sure to also:
 
-1. Edit the `DESCRIPTION` as appropriate with the correct package information;
-
-2. Edit the files in `R/`, `tests/`, and `vignettes/` to suit your package;
-
-3. Remove these instructions from `README.Rmd`, and re-render the `.md` file using `devtools::render_readme()`.
+_daedalus.data_ prepares data objects to be used by the _daedalus_ (see [here](https://github.com/jameel-institute/daedalus)) model.
 
 ## Installation
 
-**NOTE:** Remove or comment out installation sources as appropriate.
-
-You can install the development version of ji.rpkg.template from the Jameel Institute R-universe with:
+You can install the development version of _daedalus.data_ from the Jameel Institute R-universe with:
 
 ```r
-# installation from R-universe
-# install.packages(
-#   "ji.rpkg.template", 
-#   repos = c(
-#     "https://jameel-institute.r-universe.dev", "https://cloud.r-project.org"
-#   )
-# )
+installation from R-universe
+install.packages(
+ "daedalus.data",
+ repos = c(
+   "https://jameel-institute.r-universe.dev", "https://cloud.r-project.org"
+ )
+)
 ```
 
 or from GitHub [GitHub](https://github.com/) with:
 
 ``` r
-# install.packages("pak")
-# pak::pak("jameel-institute/ji.rpkg.template")
+install.packages("pak")
+pak::pak("jameel-institute/daedalus.data")
 ```
 
-## Quick start
+### Installation notes
 
-Add a simple example of using the package's main feature(s) here, with a minimum amount of code.
-If preparatory or plotting steps are needed, prefer to hide them to keep focus on the package functionality.
+Both _daedalus.data_ and _daedalus_ are under active development. To use them, alwasy ensure you have the latest versions on main branch.
 
-## Related projects
 
-Add information and links to related projects, such as research papers or packages, here.
+## Quick start
```

Comment:
In this section, it might be good to just print the output of `data(package = "daedalus.data")` to show the data names. Other data packages with fewer datasets often explain each one, but we have too many for that.